UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common.UnitTests/HealthCheck/AzureWebJobHealthCheckTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿
+using FakeItEasy;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using UKHO.ExternalNotificationService.Common.HealthCheck;
+
+namespace UKHO.ExternalNotificationService.Common.UnitTests.HealthCheck
+{
+    public class AzureWebJobHealthCheckTest
+    {
+        private IAzureWebJobHealthCheckService _fakeAzureWebJobHealthCheckService;
+        private AzureWebJobHealthCheck _azureWebJobHealthCheck;
```

Comment:
Should this be `IHealthCheck` instead of `AzureWebJobHealthCheck`?

Reply:
I think it will be fine to use AzureWebJobHealthCheck. can refer to another health check class at below path. 

Tests >>UKHO.ExternalNotificationService.Common.UnitTests>>HealthCheck